### ISSUE TITLE
Chef Metal with 1.0

### ISSUE THREAD

**Issue by uchit66** [chef/chef-provisioning#396]
Dear Team,

When Chef Metal will be come out with 1.0 version for Production ready. From versioning, its looks it is in Beta. So is it really in beta or ready for Production use?

**Comment by tyler-ball** [comment]
Hey @uchit66 - `chef metal` was renamed to `chef provisioning`.  The core `chef-provisioning` software is 1.0 and ready for production use, but there are still issues and bugs that we are actively tracking down and closing.  I would highly suggest using it in a test environment first and making sure these bugs do not affect you, or wait until they are fixed to use it in production.

The specific drivers (like `chef-provisioning-aws` and `chef-provisioning-vagrant`) have their own versions.  Generally, anything that is greater than 1.0 is production ready (it has a supported API) but again, it may have active bugs that we are working on.

TL;DR - use it in a test environment first.  But we do have people successfully using it in production.
